### ISSUE TITLE
Can not install nexe

### ISSUE THREAD

**Issue by tresdin** [jaredallard/nexe#172]
I got this error while I was trying to install `nexe` via this command `npm install nexe -g`.
```
npm ERR! Linux 3.19.0-41-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "nexe" "-g"
npm ERR! node v5.5.0
npm ERR! npm  v3.3.12
npm ERR! path /usr/lib/node_modules/nexe/bin/nexe
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/lib/node_modules/nexe/bin/nexe'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /home/project/npm-debug.log

**Comment by reverentgeek** [comment]
Same error on Mac OS X El Capitan.

**Comment by jaredallard** [comment]
This may be an issue with npm. I'll look into it right now.

**Comment by jaredallard** [comment]
Reproduced on Linux. Interesting.

**Comment by jaredallard** [comment]
Fixed by nexe@1.0.8 (3f997ca it would seem)

For some reason npm wasn't including bin/nexe.
